src/styles/global.css

```
@@ -134,3 +178,103 @@
+     .font-orbitron class overrides — the nav uses .cetus-wordmark). */
+  --cetus-display-font: 'Fraunces', serif;
+  --cetus-display-weight: 300;
+  --cetus-display-style: italic;
+  --cetus-wordmark-bg: none;
+  --cetus-wordmark-fill: var(--cetus-ink);
+  --cetus-wordmark-animation: none;
+  /* Bind font-mono to JetBrains Mono under the abyssal scope so the hero
+     kicker + catalog mono labels match SpecimenCard's scoped 'JetBrains Mono'.
+     Default pages keep Tailwind's system mono stack (no regression). */
+  --font-mono: 'JetBrains Mono', ui-monospace, SFMono-Regular, Menlo, monospace;
+  background: var(--cetus-page-bg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)